google-maps/e2e-tests/android/app/build.gradle

```diff
@@ -1,21 +1,5 @@
-plugins {
```

Comment:
Was this supposed to be removed? Without it the Android app will throw a manifest error.